capstone/src/main/java/com/google/univiz/api/SearchResourceImpl.java

```diff
@@ -0,0 +1,19 @@
+package com.google.univiz.api;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** Implementation of SearchResource Interface */
```

Comment:
This javadoc currently does not add any value. it's documenting something that is pretty obvious. Either remove it or add more details.

Reply:
removed!